lisa/tests/test_testsuite.py

```diff
@@ -196,79 +195,79 @@ def test_test_result_canrun(self) -> None:
             else:
                 self.assertEqual(False, result.can_run)
 
-    def test_skip_before_suite_failed(self) -> None:
+    async def test_skip_before_suite_failed(self) -> None:
```

Comment:
Is the `unittest` framework designed to work with asyncio?  (I really need to read more about asyncio in Python. My head is still stuck in 2.7.)

Reply:
Well...it can be made so with `IsolatedAsyncioTestCase` but ya the base case is no, it doesn't, and worse, doesn't report that it's failing. Working on it.

Reply:
Fixed.